src/public/services/__tests__/AdminSubmissionsService.test.ts

```diff
@@ -162,4 +175,102 @@ describe('AdminSubmissionsService', () => {
       )
     })
   })
+
+  describe('downloadAndDecryptAttachment', () => {
+    const MOCK_URL = 'www.decryptme.com'
+    const MOCK_SECRET_KEY = 'this is a secret'
+    const MOCK_PUBLIC_KEY = 'some public key'
+    const MOCK_NONCE = 'use me once and throw away pls'
+    const MOCK_BINARY =
+      '01101001 00100000 01100001 01101101 00100000 01100001 00100000 01101000 01110101 01101101 01100001 01101110'
+
+    it('should decrypt successfully when there is data', async () => {
```

Comment:
should we test for case where the sdk returns null?

Reply:
yes we really should - but because the sdk just uses `nacl.box.open`, this requires that the prior step of `decryptBase64` is not mocked. i can't seem to figure this out, which is why i've omitted this test. 

any hints towards figuring this out would be greatly appreciated!!!

Reply:
mock the sdk returning null

Reply:
yea just mock out the sdk

Reply:
very good point - we're just testing that the service method works, not the sdk. thanks for the help! will add it in!